app/models/assignment.rb

```diff
@@ -697,17 +691,6 @@ def next_criterion_position
     get_criteria.count > 0 ? get_criteria.last.position + 1 : 1
   end
 
-  # Returns the class of the criteria that belong to this assignment.
-  def criterion_class
```

Comment:
@GitGeminiano this indeed should be removed. The test script attributes should contain a `criterion_id` field that you can use to set the `criterion_type`.